inlong-sort/sort-connectors/mongodb-cdc/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>sort-connector-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>audit-sdk</artifactId>
+            <version>${project.version}</version>
```

Comment:
already exist in connetor-base

Reply:
I suggest write dependency directly. Because this module self use audit-sdk.

Reply:
Every connector will extends connector-base.

Reply:
> Every connector will extends connector-base.

short path denpendcy is better